.travis.yml

```diff
@@ -18,5 +18,8 @@ before_script:
 script:
   - cargo fmt -- --check &&
     cargo test --verbose --release
+  - if [ "${TRAVIS_OS_NAME}" = linux ]; then
```

Comment:
Seems wasteful (slower) to do it here ? Better to put it after `cargo fmt` check and before `cargo test`. It's more like head of the queue blocking where something that could complete much faster is delayed by something which is much slower to complete.